platform/ios/src/MGLMapView.h

```diff
@@ -79,6 +79,21 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingMode) {
     MGLUserTrackingModeFollowWithCourse,
 };
 
+/** Options for `MGLMapView.preferredFramesPerSecond`. */
+typedef NSInteger MGLMapViewPreferredFramesPerSecond NS_TYPED_EXTENSIBLE_ENUM;
+
+/**
+ The default frame rate. This can be either 30 FPS or 60 FPS, depending on
+ device capabilities.
+ */
+FOUNDATION_EXTERN MGL_EXPORT const MGLMapViewPreferredFramesPerSecond MGLMapViewPreferredFramesPerSecondDefault;
+
+/** A conservative frame rate; typically 30 FPS. */
+FOUNDATION_EXTERN MGL_EXPORT const MGLMapViewPreferredFramesPerSecond MGLMapViewPreferredFramesPerSecondLowPower;
```

Comment:
I wonder if `lowPower` would be better for a even-lower setting. 30fps is still be pretty decent frame rate. Did you run any energy tests?

Reply:
I’m not entirely happy with `lowPower` as the name for this setting, but I couldn’t think of anything better. Google calls this mode `conservative`, which avoids the potential confusion of “does this setting use less energy or is it meant for underpowered devices?” — both are true, I suppose.